packages-web/badge/src/components/Badge.tsx

```diff
@@ -0,0 +1,35 @@
+import { createElement, ReactElement } from "react";
+import classNames from "classnames";
+
+import "../ui/Badge.css";
+
+export interface BadgeProps {
+    type: "badge" | "label";
+    className?: string;
+    style?: object;
+    value?: string;
+    bootstrapStyle?: BootstrapStyle;
+    clickable?: boolean;
+    onClick?: () => void;
+    getRef?: (node: HTMLElement) => void;
```

Comment:
Ref is not being used anywhere. just remove it

Reply:
Removed it